test/e2e/cluster_upgrade.go

```diff
@@ -262,6 +264,8 @@ var _ = Describe("Skipped", func() {
 				expectNoError(validate(f, svcName, rcName, ingress, replicas))
 				By("Performing a master upgrade")
 				testMasterUpgrade(ip, v, masterUpgrade)
+				By("Checking master version")
```

Comment:
Another good question. Kube-push uses local binaries, so I don't think the whole versioning situation is as well defined as passing an explicit version (as in all other non-kube-push cases). We might be able to shell out and check the latest git commit in the user's repo (assuming this test is running in a git repo), but I opted to just address only "official" upgrade paths in this PR. Is this OK?